policyengine_us/variables/gov/hhs/chip/is_chip_eligible_child.py

```diff
@@ -0,0 +1,52 @@
+from policyengine_us.model_api import *
+
+
+class is_chip_eligible_child(Variable):
+    value_type = bool
+    entity = Person
+    label = "Child eligible for CHIP"
+    documentation = "Determines if a child is eligible for the Children's Health Insurance Program"
+    definition_period = YEAR
+    reference = (
+        "https://www.ssa.gov/OP_Home/ssact/title21/2110.htm",
+        "https://www.medicaid.gov/medicaid/national-medicaid-chip-program-information/medicaid-childrens-health-insurance-program-basic-health-program-eligibility-levels",
+    )
+
+    def formula(person, period, parameters):
+        # Get state code
+        state = person.household("state_code_str", period)
+
+        # Check age eligibility
+        age = person("age", period)
+        p = parameters(period).gov.hhs.chip.children
+        min_age = p.min_age[state]
+        max_age = p.max_age
+
+        is_age_eligible = (age >= min_age) & (age <= max_age)
+
+        # Check if state offers CHIP program for children
+        income_limit = p.income_limit[state]
+        state_has_chip = income_limit > 0
```

Comment:
Are we representing the presence of CHIP as income_limit > 0 

If so I would probably prefer mirroring the legal code and having a boolean parameter whether chip is available in a given state

Reply:
same suggestion for all variable files

Reply:
I am ok with filing a separate issue but we should represent the availability of certain parts of medicaid / chip through boolean parameters - closer to the law 
@MaxGhenis